OutOfSchool/OutOfSchool.WebApi/Controllers/V1/WorkshopMementoController.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using OutOfSchool.BusinessLogic.Services.Memento.Interfaces;
+using OutOfSchool.BusinessLogic.Services.Memento.Models;
+
+namespace OutOfSchool.WebApi.Controllers.V1;
+
+/// <summary>Controller with CRUD operations for RequiredWorkshopMemento entity.</summary>
+[ApiController]
+[ApiVersion("1.0")]
+[Route("api/v{version:apiVersion}/[controller]/[action]")]
+public class WorkshopMementoController : BaseMementoController<RequiredWorkshopMemento>
+{
+    /// <summary>Initializes a new instance of the <see cref="WorkshopMementoController" /> class.</summary>
+    /// <param name="mementoService">The memento service.</param>
+    public WorkshopMementoController(
```

Comment:
(Code style) Primary constructor could be used here

Reply:
It's done.
Used the primary constructor in the WorkshopMementoController.